rholang/src/main/scala/coop/rchain/rholang/interpreter/Splittable.scala

```diff
@@ -22,4 +26,22 @@ object Splittable {
       Splittable[F].takeFirst[A](self)
   }
 
+  implicit def splittableInd[F[_], G[_]](
```

Comment:
don't you feel that this needs a few words of explanation?

Reply:
Glad you've asked. It probably does then :) What part is unclear? The induction? The required `FunctorLayerFunctor` typeclass?

Reply:
I think this should suffice. Ping me if not, I'll add what's needed in a further PR :)